roles/prometheus/defaults/main.yml

```diff
@@ -96,6 +96,14 @@ prometheus_services:
     scrape_target: no
     volumes:
      - "/etc/prometheus/snmp-exporter:/etc/snmp_exporter"
+  ceph-exporter:
```

Comment:
The prometheus role already has support for looping over all the services defined here, and in the event of there being no `image` or `service_name` defined, it will simply add entries to the prometheus configuration to scrape these hosts on this port.